adapters/repos/db/vector/hnsw/search.go

```diff
@@ -370,10 +367,10 @@ func (h *hnsw) searchLayerByVectorWithDistancer(ctx context.Context,
 							connectionsReusable[realLen] = expId
 							realLen++
 						} else if hop < maxHops {
-							visitedExp.Visit(expId)
```

Comment:
Ahh so this previously stopped the additional hops due to the node id already being visited?